src/components/VencordSettings/UpdaterTab.tsx

```diff
@@ -182,9 +183,7 @@ function Newer(props: CommonProps) {
     );
 }
 
-function Updater() {
-    const settings = useSettings(["notifyAboutUpdates", "autoUpdate", "autoUpdateNotification"]);
-
+function updaterBody() {
```

Comment:
React components should be named in PascalCase

```suggestion
function UpdaterBody() {
```